main.tf

```diff
@@ -1,10 +1,32 @@
-/*
-eks cluster
-*/
+########################
+### eks cluster
+########################
+locals {
+  name                                    = coalesce(var.name, "eks-conf")
+  cluster_name                            = coalesce(var.cluster_name, "complete-eks-cluster")
```

Comment:
I think the name of the cluster is not `var.cluster_name`, but `aws_eks_cluster.main.name`. I could be wrong, but take a look at this in my deployed example:

**_examples/cluster/main.tf_**
```
... 
locals {
  name               = "boldlink-test"
  cidr_block         = "10.0.0.0/16"
  tag_env            = "dev"
  cluster_name       = "dynamic-eks-cluster"
... 
```

But the name of the actual Kubernetes cluster was `complete-eks-cluster`:
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/7386479/165820608-9e4c627b-47f8-424e-8eb1-c90f20f76715.png">

Reply:
This is solved by adding `cluster_name` when creating the VPC module